pages/common/git-mv.md

```diff
@@ -0,0 +1,11 @@
+# git mv
+
+> Move or rename files and update the git index.
+
+- Rename or move file and add it to next commit
+
+`git mv {{filename}} {{new_filename}}`
+
+- Rename or move file, even if target exists
```

Comment:
I would prefer this to explicitly say "overwrite the file in the target path if it exists", to ensure people understand what they're doing.